### DISCUSSION TITLE
Nuxt error pinia after command yarn start

### DISCUSSION THREAD

**Discussion by alessioraggioli** [vuejs/pinia#2034] [Help and Questions]
### Reproduction

prod

### Steps to reproduce the bug

`{
"private": true,
"overrides": {
"vue": "latest"
},
"scripts": {
"dev": "nuxt dev --port 4100",
"build": "nuxt build",
"start": "nuxt start",
"generate": "nuxt generate",
"preview": "nuxt preview",
"postinstall": "nuxt prepare"
},
"devDependencies": {
"@nuxtjs/apollo": "^5.0.0-alpha.5",
"@nuxtjs/google-fonts": "^3.0.0-0",
"@nuxtjs/moment": "^1.6.1",
"nuxt": "^3.2.1",
"sass": "^1.55.0",
"sass-loader": "10",
"vite-plugin-fonts": "^0.6.0"
},
"dependencies": {
"@mdi/font": "^7.0.96",
"@pinia/nuxt": "^0.4.7",
"fs.promises": "^0.1.2",
"pinia": "^2.0.32",
"pinia-plugin-persistedstate": "^2.3.0",
"vuetify": "^3.0.0"
}
}`

### Expected behavior

builded version is not working.

### Actual behavior

Hello,
seems like pinia is working only in dev mode.
After the command yarn start, I receive a compilation error

### Additional information

Cannot find module 'C:\Users\Alessio\Documents\nuxt3-gfl-base-template.output\server\node_modules\pinia\dist\pinia' imported from C:\Users\Alessio\Documents\nuxt3-gfl-base-template.output\server\chunks\config.mjs

**Comment by nhatimme** [comment]
Is there any fix? We got the same problems. Very annoying that the production version does not work with Pinia now.....

**Comment by nhatimme** [comment]
@danielroe Maybe you can help us out?
